docker/containers/docker_dev/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG SOURCE=ghcr.io/neuralmagic/cuda-python3.8
+# how were these versions selected before?
+ARG TORCH_VERSION=1.10.2
+ARG TORCHVISION_VERSION=0.11.3
```

Comment:
do you think we should be installing torchvision by default now? I don't think it's too heavy but worth weighing with the LLM focus

Reply:
I think potentially start with and move as needed?